sw/device/silicon_creator/mask_rom/mask_rom.c

```diff
@@ -27,6 +28,9 @@
 
 #include "hw/top_earlgrey/sw/autogen/top_earlgrey.h"
 
+// FIXME: Linker configuration
```

Comment:
Maybe we should add an issue for this? What exactly do we need to fix?

Reply:
Created #7325.